packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -378,13 +404,26 @@
           {chainNameQuery.name || `Unknown chain #${walletChain?.id}`}
           <Text color="secondaryText" size="xs">
             <AccountBalance
-              fallbackComponent={<Skeleton height="1em" width="100px" />}
-              loadingComponent={<Skeleton height="1em" width="100px" />}
-              formatFn={(num: number) => formatNumber(num, 9)}
+              fallbackComponent={<Skeleton height="1em" width="70px" />}
+              loadingComponent={<Skeleton height="1em" width="70px" />}
+              chain={walletChain}
+              tokenAddress={
+                props.displayBalanceToken?.[Number(walletChain?.id)]
+              }
+              formatFn={(props: AccountBalanceInfo) =>
+                formatAccountTokenBalance({ ...props, decimals: 9 })
+              }
+            />{" "}
+            <AccountBalance
+              loadingComponent={<Skeleton height="1em" width="30px" />}
               chain={walletChain}
               tokenAddress={
                 props.displayBalanceToken?.[Number(walletChain?.id)]
               }
+              formatFn={(props: AccountBalanceInfo) =>
+                ` (${formatAccountFiatBalance({ ...props, decimals: 3 })})`
+              }
+              showInFiat="USD"
```

Comment:
actually can we remove this for now?

looks a bit cramped in the connect button, and worried its gonna kill the pay server once we ship this because everyone using the connect button will start making those requests.

I think it should be an option on the ConnectButton to display fiat values, and would like for @jakubkrehel to do a design pass on it

Reply:
Added an option for the ConnectButton to display diat value, called `showBalanceInFiat`. Example added to changeset